src/wrap.c

```diff
@@ -1954,39 +1954,60 @@ getSettings(bool attachedFlag)
     return settings;
 }
 
-static char *
-fileNameFromAddr(uint64_t addr)
+/*
+ * Is the function func a system function?
+ * We test this by determining if the function exists in a system library.
+ * We start with checking libc. May need to add additional libs?
+ */
```

Comment:
are we limiting ourselves to only accepting functions in libc as being legitimate?

Reply:
yes. we are for now. needs to be extended...

Reply:
Add to the comment: If at this point we are still concerned with the function, and IF it is a system function (seen in libc), then we will notify about it.

Reply:
added clarification to the comment.